Modules/IO/GDCM/test/CMakeLists.txt

```diff
@@ -163,3 +164,12 @@ itk_add_test(NAME itkGDCMImageIONoPreambleTest
   COMMAND ITKIOGDCMTestDriver itkGDCMImageIONoPreambleTest
   DATA{Input/NoPreambleDicomTest.dcm}
   )
+
+itk_add_test(NAME itkGDCMImageReadWriteTest_RGB
```

Comment:
Perhaps add one more test here, to establish the pattern of how to add more tests?